app/models/locker_application.rb

```diff
@@ -79,4 +79,15 @@ def self.migrate_accessible_field
       application.update(accessibility_needs: needs)
     end
   end
+
+  def duplicates
+    LockerApplication.where(user:)
+                     .where(complete: true)
+                     .where(archived: false)
+                     .where('NOT EXISTS (:locker_assignments)',
```

Comment:
I start getting lost here - would it be possible to add a comment to the code to clarify this block? 

It seems like it's saying "Include the locker application if there is not an associated locker assignment that does not have a released_date" (but I'm not yet clear on the released_date bit)